src/controllers/post.js

```diff
@@ -1,4 +1,7 @@
 import { sendDataResponse } from '../utils/responses.js'
+import { PrismaClient } from '@prisma/client'
+import jwt from 'jsonwebtoken'
+const prisma = new PrismaClient()
```

Comment:
don't create a new prisma client, import the globally available prisma client in `/utils/dbClient.js`